elements/urth-core-channels/urth-core-channels.html

```diff
@@ -105,9 +105,11 @@
             console.debug('urth-core-channels onModelChange', options);
 
             var changes = options.changed;
-            Object.keys(changes).forEach(function(change){
+            Object.keys(changes).filter(function(change){
+                return !(change === "status");
```

Comment:
Don't like this because it seems that the `status` name is now taken and cannot be used in a template as a variable name. I wonder if to avoid this we should use a custom message and see if we can trap that on the browser side.